.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         php-version:
           - "8.2"
           - "8.3"
+          - "8.4"
```

Comment:
Can we also ensure PHP 8.4 is actually allowed in composer.json: https://github.com/Lendable/composer-license-checker/blob/967f594e79e09ce49ca7593cfe2dada8d652b5ad/composer.json#L13